src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -92,38 +91,6 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
     }
   }
 
-  protected def sourceRelations(df: DataFrame): Seq[Relation] =
```

Comment:
Could you revert this refactoring if it's not required for this PR? The PR itself is getting bigger and let's try to stabilize it.

Reply:
Reverted. Thanks!